api/pom.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>no.nav.soknad.arkivering</groupId>
+		<artifactId>soknadsfillager</artifactId>
+		<version>1.1.0</version>
+	</parent>
+
+	<artifactId>fillager-api</artifactId>
+	<name>${project.artifactId}</name>
+	<description>API for soknadsfillager</description>
+
+	<properties>
+		<spring-web.version>5.3.13</spring-web.version>
+		<javax-servlet-api.version>2.5</javax-servlet-api.version>
+		<javax-validation-api.version>2.0.1.Final</javax-validation-api.version>
+		<openapi-generator-maven-plugin.version>5.3.0</openapi-generator-maven-plugin.version>
+
+		<springfox.version>3.0.0</springfox.version>
+		<jackson-databind-nullable.version>0.2.2</jackson-databind-nullable.version>
+
+		<kotlin.version>1.5.20</kotlin.version>
+		<java.version>11</java.version>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-web</artifactId>
```

Comment:
Why the API package needs spring-web dependency ? I thought the controller is in the other package ?

Reply:
The generated code won't build without the dependency.